### PR TITLE
Fix delete metadata update

### PR DESCRIPTION
**Pull request by annymsMthd**
Signed-off-by: Joshua Benjamin <joshuabnjmn@gmail.com>

### CODE REVIEW

**Comment by annymsMthd** [comment]
Npgsql doesn't like multiple commands in the delete for some reason.

**Comment by CumpsD** [comment]
Thanks @annymsMthd

**Comment by Aaronontheweb** [comment]
crap, I need to update the CI system for this plugin, don't I?

**Comment by CumpsD** [comment]
and all other PRs ;)

**Comment by Arkatufus** [comment]
@CumpsD Can you resolve the merge conflict?

**Comment by Arkatufus** [comment]
@annymsMthd Can you resolve the merge conflict?

File: src/Akka.Persistence.PostgreSql/Journal/PostgreSqlQueryExecutor.cs [hunk 1]
```diff
@@ -93,6 +103,8 @@ public PostgreSqlQueryExecutor(PostgreSqlQueryConfiguration configuration, Akka.
         protected override DbCommand CreateCommand(DbConnection connection) => ((NpgsqlConnection)connection).CreateCommand();
         protected override string CreateEventsJournalSql { get; }
         protected override string CreateMetaTableSql { get; }
+        protected override string DeleteBatchSql { get; }
+        public string DeleteBatchSqlMetadata { get; }
```
Comment (Arkatufus) [inline src/Akka.Persistence.PostgreSql/Journal/PostgreSqlQueryExecutor.cs:107]: This property should be made protected.

File: src/Akka.Persistence.PostgreSql/Journal/PostgreSqlQueryExecutor.cs [hunk 2]
```diff
@@ -54,6 +56,14 @@ public PostgreSqlQueryExecutor(PostgreSqlQueryConfiguration configuration, Akka.
                     {Configuration.SequenceNrColumnName} BIGINT NOT NULL,
                     CONSTRAINT {Configuration.MetaTableName}_pk PRIMARY KEY ({Configuration.PersistenceIdColumnName}, {Configuration.SequenceNrColumnName})
                 );";
+            
+            DeleteBatchSql = $@"
+                DELETE FROM {Configuration.FullJournalTableName}
+                WHERE {Configuration.PersistenceIdColumnName} = @PersistenceId AND {Configuration.SequenceNrColumnName} <= @ToSequenceNr;";
+
+            DeleteBatchSqlMetadata = $@"
```
Comment (Arkatufus) [inline src/Akka.Persistence.PostgreSql/Journal/PostgreSqlQueryExecutor.cs:64]: Can you add a comment here, explaining why we need to separate the SQL script into two?

**Review by Arkatufus — CHANGES_REQUESTED** [review]
Looks good, just needed a few tweaks.

File: src/Akka.Persistence.PostgreSql/Journal/PostgreSqlQueryExecutor.cs (hunk 1 above)
Comment (CumpsD) [reply src/Akka.Persistence.PostgreSql/Journal/PostgreSqlQueryExecutor.cs:107]: Or private?

**Pull request closed without merging**